### DISCUSSION TITLE
Some suggestions about Epics

### DISCUSSION THREAD

**Discussion by funcode** [ankitpokhrel/jira-cli#631] [Ideas]
1. Is it possible to filter/list/view epics by the "epic name"?
2. Show "Issues without epics" 
3. Add flags to show issues in Epics that are not in "Done" status. "jira epic list -s~Done" shows epics, not issues that are not "Done".

**Comment by funcode** [comment]
Used --jql to achieve the goal.

**Reply by LordReydrX** [reply]
If your environment is configured with an implementation of ScriptRunner you can do some pretty deep queries.
